src/Alert/Alert.js

```diff
@@ -9,6 +9,13 @@ import CloseIcon from '../icons/CloseIcon'
 
 import { colors } from '../variables'
 
+// We need this on every component to apply our minimal base
+// This runs on top of normalize.css which is included here in the storybook
+// template, we don't want to call it from here since we don't want it included
+// on every component
+import globalStyles from '../styles.global'
+globalStyles()
```

Comment:
will this be injecting this to the top level once we import it into the app? (like could it bleed to other parts?) the global styles are defined on the body (like font-size, line-height, etc.)